packages/components/src/segmented-control/stories/index.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	alignCenter,
+	alignJustify,
+	alignLeft,
+	alignRight,
+} from '@wordpress/icons';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Icon, __experimentalSpacer as Spacer } from '../../';
+import { SegmentedControl } from '../index';
+import { View } from '../../view';
+
+export default {
+	component: SegmentedControl,
+	title: 'Components/SegmentedControl',
+};
+
+const align = [
```

Comment:
In storybook there is one violation with this `ARIA attribute is not allowed: aria-selected="true"`, which I couldn't figure out.. --cc @diegohaz